doc/services/zbus/index.rst

```diff
@@ -1,58 +1,74 @@
 .. _zbus:
 
-Zephyr message bus (zbus)
-#########################
+Zephyr Bus (ZBus)
+#################
 
-The :dfn:`Zephyr message bus - Zbus` is a lightweight and flexible message bus enabling a simple way for threads to talk to one another.
+The :dfn:`Zephyr Bus - ZBus` is a lightweight and flexible software bus enabling a simple way for threads to talk to one another.
 
 .. contents::
     :local:
     :depth: 2
 
 Concepts
 ********
-
-Threads can broadcast messages to all interested observers using zbus. Many-to-many communication is possible. The bus implements message-passing and publish/subscribe communication paradigms that enable threads to communicate synchronously or asynchronously through shared memory. The communication through zbus is channel-based, where threads publish and read to and from using messages. Additionally, threads can observe channels and receive notifications from the bus when the channels are modified. The figure below shows an example of a typical application using zbus in which the application logic (hardware independent) talks to other threads via message bus. Note that the threads are decoupled from each other because they only use zbus' channels and do not need to know each other to talk.
+Threads can broadcast messages to all interested observers using ZBus. Many-to-many communication is possible. The bus implements message-passing and publish/subscribe communication paradigms that enable threads to communicate synchronously or asynchronously through shared memory. The communication through ZBus is channel-based, where threads publish and read to and from using messages. Additionally, threads can observe channels and receive notifications from the bus when the channels are modified. The figure below shows an example of a typical application using ZBus in which the application logic (hardware independent) talks to other threads via software bus. Note that the threads are decoupled from each other because they only use ZBus' channels and do not need to know each other to talk.
```

Comment:
documentation should also following coding guidelines and have lines wrap at 100 characters. 
The long lines are making the actuall differences hard to review :/

Reply:
Do you suggest any formatting tool like `clang-format` to do this? I am trying to use the [rstfmt](https://pypi.org/project/rstfmt/), but it is not working because of the sphinx commands. I suggest standardizing that like we do with the code. I will try out something here to improve the documentation format and organization.

Reply:
The old and good Neovim (Lazyvim, in my case), with its `gw` keybinding, immensely helped.

Reply:
The only issue here is that the new line-split file will also be hard to keep track of because it will change all the text.

Reply:
Just make a separate commit for the formatting-only changes?

Reply:
Ok. I will do.

Reply:
It will not be possible. There are tons of rework to do. I tried to do that, but there were many conflicts and other issues. I am rewriting a significant portion of that. It may take more time to review, but it would be better to check all to see if it is still consistent. Sorry for the burden.